planner/core/preprocess.go

```diff
@@ -295,7 +296,7 @@ func (p *preprocessor) checkAutoIncrement(stmt *ast.CreateTableStmt) {
 		}
 	}
 	if (autoIncrementMustBeKey && !isKey) || count > 1 {
-		p.err = errors.New("Incorrect table definition; there can be only one auto column and it must be defined as a key")
+		p.err = autoid.ErrWrongAutoKey.GenWithStackByArgs()
```

Comment:
If arguments is empty, I think we can use `GenWithStack()`

Reply:
@zz-jason `GenWithStack()` accepts a formatted string, but there is no such string here.